### PR TITLE
aeAccountBackground: A component as background for identity related elements

### PR DESCRIPTION
**Pull request by acdbaykal**
[](https://postimg.org/image/p2vj4dg2j/)

### CODE REVIEW

**Comment by davidyuk** [comment]
AePanel, AeBanner, and AeButton technically is also a rounded container. Maybe here can be used a more particular name? Like AeCard or AeAccountBackground.

File: src/components/aeRoundedContainer/index.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import aeIdentityAvatar from './aeRoundedContainer.vue'
```
Comment (davidyuk) [inline src/components/aeRoundedContainer/index.js]: `aeIdentityAvatar` should be replaced with `aeRoundedContainer`

Comment (acdbaykal) [reply src/components/aeRoundedContainer/index.js]: Whoops show did that slip in :(

**Comment by acdbaykal** [comment]
I like AeAccountBackground. It's not always used like a card.

File: src/components/aeRoundedContainer/aeRoundedContainer.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+      default: 'stretch',
+      validator: value => ['start', 'end', 'center', 'baseline', 'stretch'].includes(value)
+    },
+    alignContent: {
+      type: String,
+      default: 'stretch',
+      validator: value => [
+        'start', 'end', 'center', 'space-between', 'space-around', 'stretch'
+      ].includes(value)
+    }
```
Comment (davidyuk) [inline src/components/aeRoundedContainer/aeRoundedContainer.js]: Even in `scoped` mode, it is possible to set styles of the root node of child components, Is it necessary to wrap possible css properties with vue component properties with so lot of code?

File: src/components/aeRoundedContainer/aeRoundedContainer.vue [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<template>
+  <span :class="cssClasses">
```
Comment (davidyuk) [inline src/components/aeRoundedContainer/aeRoundedContainer.vue]: I think `div` has a better semantic for something with `display: flex`.

**Pull request merged**